test/integration/vm-integration-run

```diff
@@ -38,6 +38,11 @@ kpatch_dependencies
 kpatch_separate_disk_cache /dev/vdb /mnt/build
 kpatch_set_ccache_max_size 10G
 
+if [[ -d "${HOME}/src" && -f "${HOME}/src/version" ]]; then
+  cp "${HOME}/src/version" "${HOME}/.kpatch/"
+  mv "${HOME}/src" "${HOME}/.kpatch/"
+fi
```

Comment:
I'm confused about the purpose of this.  The commit message says something about Ubuntu needing the kernel sources pre-downloaded because they don't keep older versions of the kernel.  But can't those be part of the vagrant image?

Or if this is needed, a comment would help.

Reply:
Ok, makes sense.  So it has nothing to do with ubuntu specifically then?  The commit message confused me.

Reply:
Yep, not ubuntu-specific, it was just mentioned in the commit message because it is the only one I am currently using it in, others have their sources available online and are downloaded using standard kpatch-build means.